luthier/config/src/utils/dimensionUtils.lua

```diff
@@ -14,6 +14,7 @@ local M = {}
 
 local misc = require 'utils/misc'
 
+local default_type = "KeyValueStoreDimensionFactory"
```

Comment:
Same here. I would make this `KeyValueStoreDimension` rather than `KeyValueStoreDimensionFactory`.

Reply:
Changed.